docs/geedocs/docsrc/answer/releaseNotes/relNotesGEE5_3_3.rst

```diff
@@ -1,258 +1,325 @@
-         * - 700
-           - Add EL6/EL7 check to RPMs
-           - Make sure that RPMS are installed on same EL version that they were produced for.
-         * - 788
-           - Search fails after transferring and publishing a database using disconnected send from the command line
-           - Re-publish the database from the web interface.
-         * - 825
-           - Geserver fails to startup fully due to conflicting protobuf library
-           - Run ``pip uninstall protobuf`` to uninstall the protobuf library installed by pip.
-
-.. |Google logo| image:: ../../art/common/googlelogo_color_260x88dp.png
-   :width: 130px
-   :height: 44px
+|Google logo|
+
+=============================
+Release notes: Open GEE 5.3.3
+=============================
+
+.. container::
+
+   .. container:: content
+
+      .. rubric:: New Features
+
+      **Ability to set default database from admin page**. For 3D databases, there
+      is an additional option in the publish dialog that allows the user to
+      specify a database as the "default". The "default" database will be served
+      to the client if the server URL is specified without a database name.
+
+      **getop can retrieve and display information while gesystemmanager is busy**.
+      Previously, ``getop`` could not display information while system manager
+      was in the middle of major calculations. Now, this restriction has been
+      removed for most calculations, such as when system manager is determining
+      which resources overlap.
+
+      **Improved memory usage reporting for caches**.
+      The memory usage estimation for the asset and
+      asset version caches has been improved and is
+      much more accurate.
+
+      **Optimized handling for task status updates**.
+      System manager's performance while handling updates about running tasks
+      has been improved. This update is experimental and can be enabled using
+      the :ref:`GraphOperations flag <graph-operations>`.
+
+      **Use Python 2.7 with EL6-based operating systems**.
+      Change EL6 to use Python 2.7 from the SCL repositories in a virtual
+      environment for increased security. EL7 and Ubuntu are unaffected.
+
+      **Additional refactoring of auto-generated code**.
+      Move ReuseOrMakeAndUpdate and ReuseOrMakeAndUpdateSubAsset from
+      auto-generated code to the storage manager.
+
+      **Miscellaneous security updates**.
+      Security updates to prevent SQL injection attacks, buffer overflow
+      attacks, and other potential risks.
+
+      **Clarified and updated documentation**.
+      Update documentation. Updates were made for LDAP, for updated
+      XMLparams changes, in the autogenerated code, and general clarifications
+      made to existing documentation. Instructions also added for building
+      on FIPS-enabled machines.
+
+      **GraphOperations flag Improved ease of use of Portable for WMS clients**.
```

Comment:
I think the words "GraphOperations flag" are a mistake on this line. That should likely be fixed in 5.3.3, though.